x/ibc/03-connection/client/cli/tx.go

```diff
@@ -0,0 +1,507 @@
+package cli
```

Comment:
Note: Significant cleanup required on this file. If there are required proofs or `json` files make sure we have documentation on how to query for that data and also make sure it is possible to pass that data via interpolation `$()`.